isis/src/base/objs/Stretch/Stretch.cpp

```diff
@@ -47,8 +47,29 @@ namespace Isis {
     p_minimum = p_lrs;
     p_maximum = p_hrs;
     p_pairs = 0;
+    p_type = "None";
   }
 
+
+  /**
+   * Constructs a Stretch object with default mapping of special pixel values to
+   * themselves and a provided name. 
+   *  
+   * @param name Name to use for Stretch 
+   */
+  Stretch::Stretch(QString name) : Blob(name, "Stretch") {
+    p_null = Isis::NULL8;
+    p_lis = Isis::LOW_INSTR_SAT8;
+    p_lrs = Isis::LOW_REPR_SAT8;
+    p_his = Isis::HIGH_INSTR_SAT8;
+    p_hrs = Isis::HIGH_REPR_SAT8;
+    p_minimum = p_lrs;
+    p_maximum = p_hrs;
```

Comment:
Is there a reason why these aren't VALID_MIN8 and VALID_MAX8?

Reply:
This is a copy of the default constructor so this is just more of a curious question for me.

Reply:
I'm not sure, @scsides do you know?